### PR TITLE
Increase performance test cluster size

### PR DESCRIPTION
**Pull request by gongmax**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup
> /kind documentation
> /kind feature

/kind hotfix

> /kind release

**What this PR does / Why we need it**:
Performance test timed out because game servers cannot be created due to lack of CPU. Increasing the test cluster size to fix it.

**Which issue(s) this PR fixes**:

Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Review by roberthbailey — APPROVED** [review]

File: build/terraform/performance/module.tf [hunk 1]
```diff
@@ -53,8 +53,8 @@ module "gke_standard_cluster" {
   location = each.value[0]
   overrideName = format("standard-performance-test-cluster-%s", replace(each.key, ".", "-"))
   releaseChannel = each.value[1]
-  machineType = "n2-standard-2"
-  initialNodeCount = 300
+  machineType = "e2-standard-4"
```
Comment (roberthbailey) [inline build/terraform/performance/module.tf:56]: This is going to double the number of cores in the test cluster. Do you have autoscaling enabled to scale the number of nodes down when it isn't in use?

File: build/terraform/performance/module.tf [hunk 2]
```diff
@@ -53,8 +53,8 @@ module "gke_standard_cluster" {
...
+  initialNodeCount = 310
```
Comment (roberthbailey) [inline build/terraform/performance/module.tf:57]: To piggyback on the above point, if we enable node autoscaling, then we should adjust the max number of allowed nodes as the initial node count wouldn't make any difference.

**Comment by roberthbailey** [comment]
As a hotfix this seems reasonable, but I'd like to make sure we don't have a bunch of idle nodes sitting around when the tests aren't running.

File: build/terraform/performance/module.tf (hunk 1 above)
Comment (gongmax) [reply build/terraform/performance/module.tf:56]: Sorry I didn't make it quite clear in the PR description. The current test cluster is already using type "e2-standard-4" actually. The code here drifted from the fact by some mistake. So the only actual change in this PR is to increase the size to 310. 
Good point to enable autoscaling. I will test and make a quick follow up PR for it.

**Pull request merged**